src/psyclone/gocean1p0.py

```diff
@@ -321,17 +295,19 @@ def gen_code(self, parent):
 
             # Look-up the loop bounds using the first field object in the
             # list
-            sim_domain = self.unique_args_arrays[0] +\
-                "%grid%subdomain%internal%"
+            api_config = Config.get().api_conf("gocean1.0")
```

Comment:
Nice. As I've said, I think it would be even better to have a named tuple so that we'd have something like `api_config.field_properties["go_grid_xstop"].fortran` or some such.

Reply:
Done.